src/VoteProxy2.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.4.24;
+
+import 'ds-token/token.sol';
+import 'ds-chief/chief.sol';
+
+contract VoteProxy2 {
+    address public cold;
+    address public hot;
+    DSToken public GOV;
+    DSToken public IOU;
+    DSChief chief;
+
+    constructor(DSChief chief_, address cold_, address hot_) {
+        chief = chief_;
+        cold = cold_;
+        hot = hot_;
+        GOV = chief.GOV();
+        IOU = chief.IOU();
+        GOV.approve(chief, uint256(-1));
+        IOU.approve(chief, uint256(-1));
+    }
+
+    modifier onlyCold() { require(msg.sender == cold); _ }
+    modifier hotOrCold() { require(msg.sender == cold || msg.sender == hot); _ }
+
+    function vote(bytes32 slate)
+        hotOrCold
+    {
+        chief.vote(slate);
+    }
+    function lock(uint256 wad)
+        hotOrCold
+    {
+        chief.lock(wad);
+    }
+    function free(uint256 wad)
+        hotOrCold
+    {
+        chief.free(wad);
+    }
+    function release(uint256 wad)
+        onlyCold
+    {
+        GOV.transfer(cold, wad);
```

Comment:
No difference in functionality, but for consistency I would suggest we keep with the memespeak:

```suggestion
        GOV.push(cold, wad);
```